components/typography/Heading.tsx

```diff
@@ -72,7 +72,7 @@ export default function Heading({
   }
 
   return (
-    <Tag id={id} className={twMerge(textColor, classNames)}>
+    <Tag id={id} className={twMerge(`${textColor} ${classNames}`)}>
```

Comment:
If I am correct, the way `twMerge` works is, it combines the class names that are passed to it as different params separated by `,` (comma).

Passing the `textColor` and `classNames` as a single hardcode merge defeats its purpose.